src/test/java/hudson/security/docker/MultiServerTest.java

```diff
@@ -23,12 +24,19 @@
 import static org.hamcrest.core.StringContains.containsString;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertThat;
+import static org.junit.Assume.assumeFalse;
+import org.junit.BeforeClass;
 
 /**
  * Tests connecting to two different servers
  */
 @LDAPTestConfiguration
 public class MultiServerTest {
+
+    @BeforeClass public static void linuxOnly() {
+        assumeFalse("Windows CI builders now have Docker installed…but it does not support Linux images", Functions.isWindows());
```

Comment:
this makes me sad as windows can do it - and I guess most devs on windows can also do it, but now the tests will all be skipped.

could you check for a JENKINS_URL or BUILD_URL in the environment so this is not skipped if docker is available and we are running in CI?

out of scope but as a comment ->  maybe passing `--platform linux/amd64` would help (or at least we could specify the platform requirement in the `DockerContainer` such that it could test that the platform was available locally rather than failing?

Reply:
> we could specify the platform requirement in the `DockerContainer` such that it could test that the platform was available locally

Not something I am about to try. I am not even sure I have Docker for Windows still running in VirtualBox (hypervisor issues or something over my head). Feel free to contribute to `docker-fixtures`, or better yet, check whether testcontainers.org solves this and offer a patch for this and other plugins. My immediate concern here is simply to get `master` green again.

> could you check for a JENKINS_URL or BUILD_URL in the environment so this is not skipped if docker is available and we are running in CI?

I guess you mean that we are _not_ running in CI. OK.

Reply:
> I guess you mean that we are not running in CI. OK.

yup - not enough negatives